app/apptesting/test_suite.go

```diff
@@ -374,7 +374,7 @@ func (s *KeeperTestHelper) BuildTx(
 // StateNotAltered validates that app state is not altered. Fails if it is.
 func (s *KeeperTestHelper) StateNotAltered() {
 	oldState := s.App.ExportState(s.Ctx)
-	s.Commit()
+	s.App.Commit()
```

Comment:
This was needed to fix "QueryStateAltered" tests, because Commit would start a new block, which would cause a state write from downtime detector.